### PR TITLE
Add experimental feature flag to dashboard assistant

### PR DESCRIPTION
**Pull request by songkant-aws**
### Description
Documentation for new feature flag to enable experimental features like text to visualization or other contextual assistant features.

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/7856

### Version
2.16

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/dashboards-assistant/index.md [hunk 1]
```diff
@@ -120,6 +120,17 @@ The following screenshot shows a saved conversation, along with actions you can
 
 <img width="700" src="{{site.url}}{{site.baseurl}}/images/dashboards/opensearch-assistant-save-notebook.png" alt="Notebooks interface with saved OpenSearch Assistant conversations">
 
+## Enabling Experimental Assistant Features
```
Comment (kolchfa-aws) [inline _dashboards/dashboards-assistant/index.md:123]:
```suggestion
## Enabling Dashboards Assistant experimental features
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @songkant-aws! Do you want to list out the available experimental features here?

**Comment by songkant-aws** [comment]
> Thank you, @songkant-aws! Do you want to list out the available experimental features here?

For now, we only rolled out text to visualization feature. Once other features are ready, we can iterate the list.

File: _dashboards/dashboards-assistant/index.md [hunk 2]
```diff
@@ -120,6 +120,17 @@ The following screenshot shows a saved conversation, along with actions you can
 
 <img width="700" src="{{site.url}}{{site.baseurl}}/images/dashboards/opensearch-assistant-save-notebook.png" alt="Notebooks interface with saved OpenSearch Assistant conversations">
 
+## Enabling Dashboards Assistant experimental features
+**Introduced 2.16**
+{: .label .label-purple }
+
+To enable experimental assistant features such as text to visualization feature or other contextual assistant features, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
```
Comment (kolchfa-aws) [inline _dashboards/dashboards-assistant/index.md:127]:
```suggestion
To enable experimental assistant features, such as text to visualization, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
```

**Review by natebower — APPROVED** [review]
LGTM with @kolchfa-aws change.

**Pull request merged**